src/app/screens/Enable/index.jsx

```diff
@@ -16,7 +16,7 @@ function Enable(props) {
     setEnabled(true);
     msg.reply({
       enabled: true,
-      remember,
+      remember: true, // remember by defaul
```

Comment:
I think it's better to set it to true in const [remember, setRemember] = useState(false).
Now it will always be overridden in case you would use "setRemember" somewhere.